tests/neg/i18020.scala

```
@@ -41,18 +41,18 @@ def barVal: Unit =
 
 // i18050
 package p {
-  package _root_ { // error
+  package _root_ { // not-reported
```

I think we should report an error here as well. (in typedPackageDef)